common/index.ts

```diff
@@ -17,4 +17,13 @@ export const PLUGIN_ID = 'opendistroSecurity';
 export const PLUGIN_NAME = 'opendistro_security';
 
 export const API_PREFIX = '/api/v1/opendistro_security';
+export const CONFIGURATION_API_PREFIX = 'configuration';
```

Comment:
Minor:
I would suggest `API_CONFIGURATION_PREFIX = API_PREFIX + 'configuration';`

So all of those 
`${API_PREFIX}/${CONFIGURATION_API_PREFIX}/sth.` can be shorten as `${API_CONFIGURATION_PREFIX}/sth.`